test/e2ev2/govProps/resume-finality.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
can we define a utility function to create the proposal json file as a temporary file?
So we don't have to keep these and we can make it flexible for any type of proposal

Reply:
actually, now gov props should be send as sdk.Msg, instead of using CLI to send, so the files will not be needed anymore